### PR TITLE
Feature/cp 135/support v5

### PR DESCRIPTION
**Pull request by d-javed**
support and clean up

### CODE REVIEW

File: nightwatch/pageAssertions.object.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ module.exports = {
...
     about_Cop_title: "About COP",
     benefits_title: "How COP helps you",
-    use_Cop_title: "Use COP",
+    use_Cop_title: "Get started",
     roadmap_title: "Roadmap",
     security_title: "Security",
   },
-  help: {
+  support: {
     title: "Help"
```
Comment (bviswadas) [inline nightwatch/pageAssertions.object.js:13]: Support title still stays "Help"?

File: src/components/__tests__/header.js [hunk 2]
```diff
@@ -21,4 +21,16 @@ describe("Header", () => {
     const testInstance = tree.root;
     expect(testInstance.findByProps({ id: "logo-image" })._fiber.memoizedProps.source).toEqual("/images/logo.png");
   });
+
+//   it("test function", () => {
+//     const tree = renderer
+//       .create(<Header/>);
+//     const testInstance = tree.root;
+// // const func = testables.showMobileButton();
```
Comment (bviswadas) [inline src/components/__tests__/header.js:29]: remove commented out  test

File: src/components/panel.js [hunk 3]
```diff
@@ -36,7 +36,6 @@ const Panel = (props) => (
         </div>
       </div>
     </div>
```
Comment (bviswadas) [inline src/components/panel.js:38]: remove unused import as well?

File: nightwatch/pageAssertions.object.js (hunk 1 above)
Comment (d-javed) [reply nightwatch/pageAssertions.object.js:13]: this will change later

File: src/components/__tests__/header.js (hunk 2 above)
Comment (d-javed) [reply src/components/__tests__/header.js:29]: will need it for next task on jira

**Pull request merged**